pkg/intel/metadata/manifest/bootpolicy/manifest.go

```diff
@@ -27,3 +32,28 @@ type Manifest struct {
 func (bpm Manifest) StructInfo() StructInfo {
 	return bpm.BPMH.StructInfo
 }
+
+func (bpm *Manifest) ValidateIBBs(firmware uefi.Firmware) error {
+	if len(bpm.SE[0].DigestList.List) == 0 {
```

Comment:
I would also put a check that len(bpm.SE) > 0
+ why are we doing only for the first element?

Reply:
Unfortunately, the explanation is covered by NDA enforced by one of CPU vendors.

Reply:
> I would also put a check that len(bpm.SE) > 0

To do not reset approvals for this tiny fix, I'll merge as is :(